contrib/completion/bash/docker

```diff
@@ -1466,7 +1466,7 @@ _docker_ps() {
 _docker_pull() {
 	case "$cur" in
 		-*)
-			COMPREPLY=( $( compgen -W "--all-tags -a --help" -- "$cur" ) )
+			COMPREPLY=( $( compgen -W "--all-tags -a --disable-content-trust --help" -- "$cur" ) )
```

Comment:
As this option defaults to `true`, you could as well complete `--disable-content-trust=false` as in the completions for `build` and `run`.